src/networking/internet/InternetCommunicator.java

```diff
@@ -0,0 +1,310 @@
+package networking.internet;
+
+/**
+ * This file contains the class InternetCommunicator which is the object that other modules
+ * use to start the communication , send the messages across the network and subscribe
+ * for messages that they should be receiving.
+ *
+ * @author Madaka Srikar Reddy
+ * 
+ */
+
+import java.io.*;
+import java.net.*;
+import java.util.HashMap;
+import networking.*;
+import networking.queueManagement.*;
+import networking.utility.*;
+import infrastructure.validation.logger.*;
+
+public class InternetCommunicator implements ICommunicator {
+
+    /** SendQueueListener that will be listening on the send Queue */
+    private SendQueueListener sendQueueListener;
+    private Thread sendQueueListenerWorker;
```

Comment:
As SendQueueListener file is going to be changed to InternetSendQueueListener update the code accordingly